docs/2-Deploying/2-Configuration.md

```diff
@@ -1240,6 +1240,73 @@ AWS STS role ARN to use for S3 authentication.
   	sts_assume_role_arn: arn:aws:iam::1234567890:role/some-s3-role
   ```
 
+##### Vulnerabilities
+
+###### `databases_path`
+
+The path specified needs to exist and fleet needs to be able to read and write to and from it. This is the only mandatory configuration needed for vulnerability processing to work.
+
+- Default value: none
+- Environment variable: `FLEET_VULNERABILITIES_DATABASES_PATH`
+- Config file format:
+
+  ```
+  vulnerabilities:
+  	databases_path: /some/path
+  ```
+
+###### `vulnerabilities_periodicity`
+
+How often vulnerabilities are checked.
+
+- Default value: `1hr`
+- Environment variable: `FLEET_VULNERABILITIES_PERIODICITY`
+- Config file format:
+
+  ```
+  vulnerabilities:
+  	vulnerabilities_periodicity: 1hr
+  ```
+
+###### `vulnerabilities_cpe_database_url`
+
+URL to fetch the CPE dictionary database from. Some users want to control where fleet gets its database from. When Fleet sees this value defined, it downloads the file directly. It expects a file in the same format as can be found in https://github.com/fleetdm/nvd/releases. If this value is not defined, Fleet checks for the latest release in Github and only downloads it if needed.
+
+- Default value: `""`
+- Environment variable: `FLEET_VULNERABILITIES_CPE_DATABASE_URL`
+- Config file format:
+
+  ```
+  vulnerabilities:
+  	vulnerabilities_cpe_database_url: ""
+  ```
+
+###### `vulnerabilities_cve_database_url`
```

Comment:
Actually, this one would be `vulnerabilities.cve_feed_prefix_url` and the env var below `FLEET_VULNERABILITIES_CVE_FEED_PREFIX_URL`. Copy+paste error!